pkg/storaged/stratis/stopped-pool.jsx

```diff
@@ -166,8 +193,18 @@ export function make_stratis_stopped_pool_page(parent, uuid) {
 }
 
 const StoppedStratisPoolCard = ({ card, uuid }) => {
-    const key_desc = client.stratis_stopped_pool_key_description[uuid];
-    const clevis_info = client.stratis_stopped_pool_clevis_info[uuid];
+    const stopped_info = client.stratis_manager.StoppedPools[uuid];
+    let key_desc = false;
+    let clevis_info = false;
+
+    if (is_v1_pool(uuid)) {
+        const kd = stopped_info.key_description;
+        if (kd && kd.v[0] && kd.v[1].v[0])
+            key_desc = kd.v[1].v[1];
+        const ci = stopped_info.clevis_info;
+        if (ci && ci.v[0] && ci.v[1].v[0])
+            clevis_info = ci.v[1].v[1];
+    }
```

Comment:
Definitely expand these and split out the array access to make it more descriptive

Reply:
Done.